Fix word

FangYuan33 opened the pull request.

FangYuan33 committed:
Fix word

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/nacos-group/nacos-group.github.io?pullRequest=991) <br/>All committers have signed the CLA.

KomachiSion reviewed (changes requested):
Please also submit changes to `next`

FangYuan33 committed:
Fix formatting of Derby database usage notes

FangYuan33 committed:
Fix typos in Derby database usage notes

FangYuan33 committed:
Clarify Derby database usage in Nacos cluster mode

Corrected wording for clarity regarding Derby database limitations in cluster mode.

FangYuan33 committed:
Fix formatting issues in Derby database notes

FangYuan33 committed:
Fix formatting issues in deployment-cluster.md

FangYuan33 committed:
Fix formatting issues in deployment-cluster.md

FangYuan33 committed:
Clarify Derby database usage in Nacos cluster mode

Corrected wording for clarity regarding Derby database limitations in cluster mode.
